### PR TITLE
proposal for guidelines

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: pages/Zotero.xml [hunk 1]
```diff
@@ -148,6 +155,10 @@ inserted in the fields: author, editor and translator.</p>
            etc.) are not needed unless their removal would mislead the
            reader.</p>
                     </item>
+                    <item>
+                     <p>In case if the transliterated form of a contributor's name in the publication differs from the transliterated form according to the rules applied by the project, enter the standardized form in accordance with http://viaf.org/ into the corresponding field.
```
Comment (PietroLiuzzo) [inline pages/Zotero.xml]: Please make that viaf url a link with the ref element and target attribute

File: pages/Zotero.xml [hunk 2]
```diff
@@ -148,6 +155,10 @@ inserted in the fields: author, editor and translator.</p>
...
+                     In addition to that, ALWAYS provide in notes the actual spelling of a contributor's name, as it appears in the publication.</p>
```
Comment (PietroLiuzzo) [inline pages/Zotero.xml]: please try to specify that the note is the one which a user can create in Zotero under Notes and not any other form of note

File: pages/Zotero.xml [hunk 3]
```diff
@@ -88,6 +88,13 @@ English.</p>
 etc.), provide the scientific transcription and translation only
 (i.e. not the title in the original script).</p>
   </item>
+  <item>
+      <p>The basis for transliteration of Cyrillic should be this table http://kodeks.uni-bamberg.de/AKSL/Schrift/Transliteration.htm</p>
```
Comment (PietroLiuzzo) [inline pages/Zotero.xml]: please encode  the link

File: pages/transliteration-principles.xml [hunk 4]
```diff
@@ -322,6 +322,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
          <p><hi rendition="simple:italic">Šarḥ sabab intiqāl mamlakat Sulaymān
          malik ʾIsrāʾīl ʾilā bilād an-naǧāšīya allatī hiya al-ḥabaša</hi></p>
        </div>
+       <div type="level2">
+         <head>Transliteration from Cyrillic</head>
+         <p>The basis for transliteration of Cyrillic should be this table http://kodeks.uni-bamberg.de/AKSL/Schrift/Transliteration.htm</p>
```
Comment (PietroLiuzzo) [inline pages/transliteration-principles.xml]: please encode the link

File: pages/transliteration-principles.xml [hunk 5]
```diff
@@ -322,6 +322,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+         <p>For more information on the transliteration rules, consult the page with Zotero guidlines.
```
Comment (PietroLiuzzo) [inline pages/transliteration-principles.xml]: you can add an internal link to the page you refer to

**Comment by DariaElagina** [comment]
Dear  Pietro,
the corrections are done.

**Pull request merged**